File: week2/day2/draw-chess-table.py

# Crate a program that draws a chess table like this
#
# % % % %
#  % % % %
# % % % %
#  % % % %
# % % % %
#  % % % %
# % % % %
#  % % % %
#

size = 8
a = 0

while a < int(size):
    a += 2
    print((int(size) // 2) * "% ")
    print((int(size) // 2) * " %")

File: week8/day2/draw-chess-table.js
'use strict';

// Crate a program that draws a chess table like this
//
// % % % %
//  % % % %
// % % % %
//  % % % %
// % % % %
//  % % % %
// % % % %
//  % % % %
//


var block = '%'
var space = ' '
var x = block + space
var y = space + block

for (var i = 0; i < (5); i++) {
    console.log(x.repeat(4));
    console.log(y.repeat(4));
}
